patch/cisco-pmbus-driver-add-support-for-additional-temp-sensor.patch

```diff
@@ -0,0 +1,110 @@
+From cfb4b5420339995a76f765e72b1c916029f264ae Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Mon, 24 May 2021 16:02:21 -0700
+Subject: [PATCH] Add supporting for additional PSU sensor in pmbus driver
+
+PSU650W has 4 temperature sensors and pmbus driver has support for 3 senosrs. Added support for 4th temp sensor (i.e. PSU Outlet temperature sensor)
```

Comment:
PSU650W sounds pretty generic. Can you please list the full model name?

Reply:
addressed in the new PR updates. Please review.